src/routing/Router.tsx

```diff
@@ -1,10 +1,24 @@
 import { createBrowserRouter, RouterProvider } from "react-router-dom"
+import { Card } from "../components/Card"
 import { routes } from "./routes"
 
 const router = createBrowserRouter([
   {
     path: routes.root(),
-    element: <div>Temporary element</div>
+    element: (
+      <div>
+        <Card
```

Comment:
Ta karta też powinna mieć color jakby zdjęcie się nie załadowało

Reply:
Racja. To może w przypadku niezaładowania zdjęcia, ustawić domyślny kolor?